specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -54,6 +67,7 @@ function ProtectedStatsPage(): JSX.Element | null {
   // REFACTOR: Make stats page component smaller.
 
   useMenuItem('statistics');
+  console.log('Change');
```

Comment:
remove console.log

Reply:
will leave it till release - not resolving right now